native/cocos/platform/android/java/src/com/cocos/lib/CocosVideoHelper.java

```diff
@@ -156,6 +160,27 @@ public void handleMessage(Message msg) {
                 helper._setVolume(msg.arg1, volume);
                 break;
             }
+            case VideoTaskSetPlaybackRate: {
+                float rate = (float) msg.arg2 / 10;
+                helper._setPlaybackRate(msg.arg1, rate);
+                break;
+            }
+            case VideoTaskSetMute: {
+                if (msg.arg2 == 1) {
```

Comment:
Why not just record boolean value for `msg.arg2`?

Reply:
Same as above.